### ISSUE TITLE
Have an example of config file

### ISSUE THREAD

**Issue by davidtys** [fbjorn/i3lock-fab#1]
Hello,

It would be great to have an example of config file to know how to config the images path.

Thanks !

**Comment by fbjorn** [comment]
Hi,

Default config file is created automatically at `~/.i3lock-fab/conf.yaml`. For me it looks like:
```
no_unlock_indicator: false
random_pic: true
show_failed_attempts: true
url: https://www.reddit.com/r/wallpaper

```
So both downloaded image and result suitable for multiple monitors are located at `~/.i3lock-fab`. Paths are hardcoded right now.

**Comment by VonKavalier** [comment]
So it's not possible for local files ?

**Comment by fbjorn** [comment]
@VonKavalier If you mean providing a pool of local images to pick from, then it's not possible. The idea of this locker is quite different: getting random pictures every time you lock the computer. It's applicable if you getting bored from one permanent background, as I do :slightly_smiling_face:

**Comment by VonKavalier** [comment]
I don't see the difference when talking about getting bored of one permanent background actually :/ I have a folder filled with 20+ images I personally selected, and would like to use them instead of images I don't have the control on :/ But if it's not the purpose of this project I get this and I will close the issue :)

**Comment by fbjorn** [comment]
Yes, the project is about uncontrolled pictures (relying on reddit users' tastes)
